app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+    def create
+      @article = Article.find(params[:article_id])
```

Comment:
Apply good practices, for example for indentation only 2 blank spaces are used
https://github.com/rubocop/ruby-style-guide

Reply:
Thank you @estebancordoba. I don't know why my editor made the indentation with 4 blank spaces